xdsl/interactive/app.py

```diff
@@ -421,18 +471,33 @@ def update_output_operation_count_tuple(self) -> None:
             case Exception():
                 self.output_operation_count_tuple = ()
             case ModuleOp():
-                self.output_operation_count_tuple = tuple(
-                    count_number_of_operations(self.current_module).items()
+                # sort tuples by count in ascending order
```

Comment:
I'd lean towards sorting them alphabetically always so that the user learns where to look for which op over time. It also means that the lists are grouped by dialect, which seems useful at first glance. I don't feel very strongly about this

Reply:
Okay.